tests/test_feeds.py

```diff
@@ -0,0 +1,229 @@
+import pytest
+from datetime import timedelta
+
+from django.http import Http404
+
+from nomination import feeds
+from . import factories
+
+
+pytestmark = pytest.mark.django_db
+
+class TestURLFeed:
+
+    @pytest.fixture
+    def url_feed_setup(self, rf):
+        request = rf.get('/')
+        project = factories.ProjectFactory()
+        feed = feeds.url_feed()
+        return request, project, feed
+
+    def test_get_object_returns_project(self, url_feed_setup):
+        request, project, feed = url_feed_setup
+        obj = feed.get_object(request, project.project_slug)
+        assert obj == project
+
+    def test_get_object_raises_404(self, url_feed_setup):
+        request, _, feed = url_feed_setup
+        with pytest.raises(Http404):
+            feed.get_object(request, '')
+
+    def test_title_returns_project_title(self, url_feed_setup):
+        _, project, feed = url_feed_setup
+        title = feed.title(project)
+        assert title == 'Latest URLs for {}'.format(project.project_name)
+
+    def test_link_returns_link_to_project_urls_feed(self, url_feed_setup):
+        request, project, feed = url_feed_setup
+        feed.get_object(request, project.project_slug)
+        link = feed.link(project)
+        assert link == '/nomination/{}/feed/urls/'.format(project.project_slug)
+
```

Comment:
If I understand the way the feed works correctly, there's never a chance for that to happen since the get_object method will fail and raise a 404 if the slug passed in the URL does not link to a project. So that particular branch should never get reached. I _think_